denizen_scripts/relay/repo-link/Handlers/message_handler.dsc

```diff
@@ -36,6 +36,8 @@ Message_Handler:
     # % ██ [ Command  Based Scripts          ] ██
       - if <[Message].starts_with[/]>:
         - choose <[Message].before[<&sp>].after[/]>:
+          - case repository repo git github:
+            - ~Run Repository_DCommand def:<list_single[<[Message]>].include[<[Channel]>|<[Author]>|<[Group]>]>
```

Comment:
For future reference and for irrelevant consistency, we should add to the bottom of the list, or we may consider alphabetizing this list based on the primary command name
We also only need to pass the `Channel` definition currently.
```suggestion
          - case repository repo git github:
            - ~Run Repository_DCommand def:<[Channel]>
```